diff -urN zxing-cpp-2.0.0/core/src/ConcentricFinder.h zxing-cpp-2.0.0.new/core/src/ConcentricFinder.h
--- zxing-cpp-2.0.0/core/src/ConcentricFinder.h	2023-01-05 16:11:06.000000000 +0100
+++ zxing-cpp-2.0.0.new/core/src/ConcentricFinder.h	2023-03-22 16:36:49.006359710 +0100
@@ -27,7 +27,7 @@
 		float b = (pattern[2] + pattern[1] + pattern[0]) / 2.f;
 		return end - (2 * a + b) / 3;
 	} else { // aztec
-		auto a = std::accumulate(&pattern[N/2 + 1], &pattern[N], pattern[N/2] / 2.f);
+		auto a = std::accumulate(pattern.begin() + (N/2 + 1), pattern.end(), pattern[N/2] / 2.f);
 		return end - a;
 	}
 }
